### DISCUSSION TITLE
Refactor features

### DISCUSSION THREAD

**Discussion by zyrain** [colgreen/sharpneat#71] [Q&A] [answered]
Hi.  Quick question.  Does this refactor support HyperNEAT, Novelty Search, or ES-HyperNEAT?  

Thanks!

**Comment by colgreen** [comment, marked as answer]
Hi. 

No. The 'sharpneat' repository supports HyperNEAT, but this refactor branch doesn't have that yet. The other ones (Novelty Search, ES-HyperNEAT) have never been implemented directly in SharpNEAT.
